Feature branch

Guy-Gustave opened the pull request:
![screen 1](https://user-images.githubusercontent.com/56753805/84577534-639ed800-adbd-11ea-819b-6a1b7f3c9652.png)
this is the navbar and 1st article

BrittanyBlake commented on css/style.css:
```diff
@@ -0,0 +1,706 @@
+  flex-flow: column;
+}
+
+/* .grid {
+  display: flex;
+  flex-flow: column;
+} */
```
Please remove commented code

BrittanyBlake commented on css/style.css:
```diff
@@ -0,0 +1,706 @@
+  margin-top: 15px;
+}
+
+/* .fund-sec-hidden {
+  display: none;
+} */
```
Please remove commented code

BrittanyBlake commented on css/style.css:
```diff
@@ -0,0 +1,706 @@
+  position: absolute;
+  width: 60px;
+  height: 60px;
+  /* z-index: 1; */
```
Please remove commented code

BrittanyBlake commented on css/style.css:
```diff
@@ -0,0 +1,706 @@
+  padding-left: 10px;
+}
+
+/* .red-img{
+
+} */
```
Please remove commented code

BrittanyBlake commented on css/style.css:
```diff
@@ -0,0 +1,706 @@
+    position: sticky;
+    }
+
+    /* .fixed-nav {
+      position: sticky;
+      margin-top: 42px;
+    } */
```
Please remove commented code

Guy-Gustave commented:
i hope i have checked all suggested. and thank you. now it is seen on mobile and large screen, well displaying

Guy-Gustave commented:
i made all suggested correction and remove all stickler errors
![article1](https://user-images.githubusercontent.com/56753805/84660944-e046c900-af19-11ea-8ca7-61b05b2c7f7b.png)

santiago-dev96 commented:
# PR under review by @santiago-rodrig (TSE)

Guy-Gustave commented:
hello rodrig, i hope you are saying the footer. but on my original page is
like this. i may align on one line but the original was like that. thank
you. and the grid that they told me that they must have  spaces i already
changed. thank you

On Mon, Jun 15, 2020 at 3:17 PM Santiago Rodriguez <notifications@github.com>
wrote:

> PR under review by @santiago-rodrig <https://github.com/santiago-rodrig>
> (TSE)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Guy-Gustave/the-next-web/pull/1#issuecomment-644129093>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANQ75DKRPBKJ5DJMO2XF54TRWYNPHANCNFSM4N5DUTVQ>
> .
>

Guy-Gustave commented:
thank you. let me focus on it i see medium have problem you're right

On Mon, Jun 15, 2020 at 3:40 PM Santiago Rodriguez <notifications@github.com>
wrote:

> *@santiago-rodrig* requested changes on this pull request.
> Changes required 🔧
>
> Good job so far! But there are still some things to do.
>
>    1. Your navigation bar is not being displayed the same way that it is
>    displayed on the original page
>
> *Original - Small Screens*
>
> [image: Screen Shot 2020-06-15 at 9 22 14 AM]
> <https://user-images.githubusercontent.com/43162770/84663016-98f51400-aeea-11ea-9049-6073fbdf5290.png>
>
> *Original - Medium Screens*
>
> [image: Screen Shot 2020-06-15 at 9 23 02 AM]
> <https://user-images.githubusercontent.com/43162770/84663189-cfcb2a00-aeea-11ea-815c-d236f2b4eed0.png>
>
> *Yours - Small Screens*
>
> [image: Screen Shot 2020-06-15 at 9 22 22 AM]
> <https://user-images.githubusercontent.com/43162770/84663228-dce81900-aeea-11ea-85f0-07852c3b99dc.png>
>
> *Yours - Medium Screens*
>
> [image: Screen Shot 2020-06-15 at 9 23 10 AM]
> <https://user-images.githubusercontent.com/43162770/84663272-ebcecb80-aeea-11ea-9a12-be5c4a1fb5a0.png>
>
>    1. Your *Latest News* section is not being displayed as it is
>    displayed on the original page
>
> *Original - Small Screens*
>
> [image: Screen Shot 2020-06-15 at 9 26 32 AM]
> <https://user-images.githubusercontent.com/43162770/84663768-98a94880-aeeb-11ea-9048-1ca482e0b745.png>
>
> *Original - Medium Screens*
>
> [image: Screen Shot 2020-06-15 at 9 26 07 AM]
> <https://user-images.githubusercontent.com/43162770/84663818-ac54af00-aeeb-11ea-89cf-51c847828896.png>
>
> *Yours - Small Screens*
>
> [image: latest_small]
> <https://user-images.githubusercontent.com/43162770/84663885-c2626f80-aeeb-11ea-80fd-c58db50704b4.gif>
>
> *Yours - Medium Screens*
>
> [image: Screen Shot 2020-06-15 at 9 25 48 AM]
> <https://user-images.githubusercontent.com/43162770/84663947-d27a4f00-aeeb-11ea-8bd1-7f3c4aed5ea9.png>
>
> Please submit another CRR after implementing the required changes, and
> please, take the time to review your project carefully, and also don't
> forget to review the page that you are trying to clone thoroughly.
>
> You can find me in slack as *srodrig*, happy coding! 👋
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Guy-Gustave/the-next-web/pull/1#pullrequestreview-430653142>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANQ75DPP3D2PQO7Q6DF67STRWYQEPANCNFSM4N5DUTVQ>
> .
>

Guy-Gustave commented:
thank you for the review. i hope i made suggested changes. thank you. can you review again?

Guy-Gustave commented:
hey Christian i hope i made almost changes, except the footer because it was like mine. when i start it was like now after i changed and now it seems like before because i changed from Saturday. another change is the latest funding, it was 4 items before. i hope the aim oh the project is now. thank you

andresporras3423 commented:
### **Pull Request Under Review By Oscar Russi (TSE)**

andresporras3423 commented on css/style.css:
```diff
@@ -0,0 +1,922 @@
+
+.tablet {
+  display: none;
+}
```
Don't repeat yourself, in order to minimize code you can group selectors with the same style definition.

andresporras3423 commented on css/style.css:
```diff
@@ -0,0 +1,922 @@
+
+.alt-img {
+  display: none;
+}
```
Same styles of selector from line 15. In order to minimize code you can group selectors with the same style definition

Guy-Gustave commented:
i finish all suggested. thank you

andresporras3423 commented:
### **Pull Request Under Review By Oscar Russi (TSE)**

andresporras3423 commented on css/style.css:
```diff
@@ -0,0 +1,918 @@
+  z-index: 2;
+}
+
+.tablet,
+.nav1,
+.nav2 {
+  display: none;
+}
```
Same styles of the selector as line 103. In order to minimize code, you can group selectors with the same style definition.

andresporras3423 commented on css/style.css:
```diff
@@ -0,0 +1,918 @@
+  width: 95%;
+}
+
+.fund-cap,
+.funding-bottom a {
+  text-decoration: none;
+}
```
Same styles of the selector as the line 274. In order to minimize code, you can group selectors with the same style definition.

andresporras3423 commented on css/style.css:
```diff
@@ -0,0 +1,918 @@
+  padding: 15px 0 5px 10px;
+}
+
+.grid-2 {
+  display: flex;
+  flex-flow: column;
+}
+
+.grid-3 {
+  display: flex;
+  flex-flow: column;
+}
```
Don't repeat yourself. In order to minimize code, you can group selectors with the same style definition.

Guy-Gustave commented:
i hope now everything is okay. even the footer i change

Guy-Gustave commented:
I don't know how it happens again I had fixed before. but check again it's okay now
